### ISSUE TITLE
Remove book image from output

### ISSUE THREAD

**Issue by tw2113** [tw2113/Goodreads-In-WordPress#2]
Try to reach out to GoodReads support to see if there's reason why there's no actual image included other than placeholder.

Remove until then.

**Comment by tw2113** [comment]
From API page

```
XML responses also include shelves and book meta-data (title, author, et cetera). The Goodreads API gives you full access to Goodreads-owned meta-data, but it does not give you full access to book meta-data supplied by third parties such as Ingram. Book cover images, descriptions, and other data from third party sources might be excluded, because we do not have a license to distribute these data via our API. 
```

**Comment by tw2113** [comment]
Closed in 8cd658ae644b0c074d21df05948337b054568024
